EIPS/eip-5173.md

```diff
@@ -0,0 +1,429 @@
+
+### Future Rewards Distribution
+
+![Figure 2: NFT Owners' Future Rewards (nFR)](../assets/eip-5173/nFR_Standard_Outline.jpeg) 
+
+*Figure 2: NFT Owners' Future Rewards (nFR)*
+
+*Figure 2* illustrates an example of a five-generation Future Rewards Distribution program based on an owner's realized profit.
+
+```solidity
+
+function _distributeFR(uint256 tokenId, uint256 soldPrice) internal virtual {
+       //...
+
+        emit FRDistributed(tokenId, soldPrice, allocatedFR);
+ }
+ 
+```
+
+The `_distributeFR` function MUST be called in the FR `transferFrom` function if there is a profitable sale. The function SHALL calculate the difference between the current sale price and the `lastSoldPrice`, then it SHALL call the `_calculateFR` function to receive the proper distribution of FR. Then it SHALL distribute the FR accordingly, making order adjustments as necessary. Then, the contract SHALL calculate the total amount of FR that was distributed (`allocatedFR`), in order to return the difference of the `soldPrice` and `allocatedFR` to the `lister`. Finally, it SHALL emit the `FRDistributed` event. 
+
+### Future Rewards Claiming
+
+The future Rewards payments SHOULD utilize a pull-payment model, similar to that demonstrated by OpenZeppelin with their PaymentSplitter contract. The event  FRClaimed would be triggered after a successful claim has been made. 
+
+```solidity
+
+function releaseFR(address payable account) public virtual override {
+        //...
+}
+
+```
+
+### Owner Generation Shifting
+
+The `_shiftGenerations` function MUST be called regardless of whether the sale was profitable or not. As a result, it will be called in the `_transfer` [EIP-721](./eip-721.md) override function and the FR `transferFrom` function. The function SHALL remove the oldest account from the corresponding `_addressesInFR` array. This calculation will take into account the current length of the array versus the total number of generations for a given token ID.
+
+## Rationale
+
+### Is This Just a Ponzi Scheme? 
+
+No, it is not. Ponzi schemes promise profits that are impossible to keep. 
+
+As opposed to fixed-yield schemes, our proposal only distributes future profits when those profits are achieved rather than guaranteeing them. Should later holders fail to make a profit, future return shares will not be distributed. 
+
+The early participants in price discovery will receive a share of profits as part of the FR implementation only and if a later owner has accumulated profits during their holdings of the token.
```

Comment:
This is more Motivation than Rationale.

The Motivation section describes the reason the EIP as a whole exists, while the Rationale section documents the reasoning behind specific technical choices within the EIP.